tests/GambitTest.php

```diff
@@ -32,8 +32,8 @@ public function testGetAllCampaigns()
         $this->assertEquals(2, $campaigns->count());
 
         // And we should be able to traverse and read values from that.
-        $this->assertEquals('Don\'t Be a Sucker', $campaigns[0]->title);
-        $this->assertEquals('Senior Story Swap', $campaigns[1]->title);
+        $this->assertEquals('World Recycle Week: Close The Loop', $campaigns[0]->title);
+        $this->assertEquals('Trash Stash', $campaigns[1]->title);
```

Comment:
Nah, that's a mockup. It doesn't matter what's there.
I just copied data from the latest json there as an example.